### ISSUE TITLE
As a reader, I want to read a statement from the editor describing the project.

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/startwords#18]
# dev notes
this is the about page, and it includes the masthead as specified in #21.

<s>this issue _also_ includes the default page summary template, because we want to show a preview of (and link to) the about page on the homepage.</s> no longer using default summary since we want to control how it displays on the homepage.

**Comment by rlskoeser** [comment]
Grant and I discussed — masthead will be included on the project about page because we want to keep things simple and reduce the number of clicks and pages.

The about page already exists on the current site as a standard hugo page, under content/about.md

**Comment by thatbudakguy** [comment]
making this a 1 so we can use it to test the default `summary.html` template - once the about page is done, we will want to test that its summary renders correctly on the homepage, so that people can get to the editorial statement/masthead/etc, and this seems like most relevant place to test that.

**Comment by thatbudakguy** [comment]
ok, this is ready for :eyes: from both @gwijthoff and @gissoo.

**Comment by gissoo** [comment]
@thatbudakguy 
This great, 
- there is the same issue with the page title "About" and the indented section at the top, where they should each shift one column to the right to match the design of the article page for mobile and tablet. Again, this is important for especially for mobile viewers because the nav at the top will be covering the content. 

May be irrelevant to you Nick, so tagging more people @gwijthoff @rlskoeser 
- This: "|"after the board of advisors is bothering me a bit, but don't feel strongly about it
- Can we revise my role so that it's "UX Designer"? I think "User Experience .." will make it long, and if people are curious they can google what "ux" means, but "Designer" is vague to me, and am not a fan.
- Do we need to call that section "Masthead"? From the titles it's clear what that section is about. I'm not quite getting its purpose, and maybe it's just me, but the term feels like representing a very print-based language.

**Comment by rlskoeser** [comment]
I also wondered about the | ! 

Would a definition list make sense here?

(Although I think these are content questions not related to the page styles)

**Comment by gwijthoff** [comment]
Just changed your title, @gissoo! 

The |'s are wholly my quirk. I love |'s. Please feel free to suggest another solution!

As for "Masthead," my thinking in using that section heading is that it signifies something about the hybrid nature of this publication. Academic journals tend to have "editorial boards," while periodicals and newspapers have "mastheads." It suggests that the writing here is built for public audiences. But I'm open to talking more about it. Thoughts @thatbudakguy @rlskoeser? If we decide we like the term "masthead," maybe the layout/design of the section heading should just change? Maybe h2 rather than h1, or insert some extra space between the end of the "about" text and the new heading?

Also, seconding @gissoo's comment on the indentation of this page's header. It's moved a little to the right on mobile, just not quite far enough.

![IMG_FFD6CDBD2ECC-1](https://user-images.githubusercontent.com/5580840/84951339-4a986e80-b0be-11ea-9db4-9021e964bbfd.jpeg)

**Comment by thatbudakguy** [comment]
> there is the same issue with the page title "About" and the indented section at the top, where they should each shift one column to the right to match the design of the article page for mobile and tablet.

shifting the content in this page's header is a little more tricky, because this page - like all non-article pages - doesn't have a concept of a "header". for the time being, I can just add some html directly into the markdown to control it, but wanted to bring it up since it's not clear what a "header" would be in the context of a non-article page. if we wanted to edit the basic page template to include a header, we could do that, but then what would go in the header? just the title? we have no way of knowing beforehand what content the page would include, so it's kind of a thorny question.

I'm fine either way about the `|`, but it seems like if we do it, shouldn't RSK and I both be listed in a list under one `|`, like the board of advisors is?

**Comment by gwijthoff** [comment]
@thatbudakguy @rlskoeser up to you both how you'd like to be described / listed!

As for the "header" question, I don't have any real preference for how it's implemented, so long as "About" and the "startword" dictionary definition aren't covered by the nav icon when initially loaded on mobile.

**Comment by gissoo** [comment]
- @thatbudakguy thanks for writing. I think for non article pages the header would mean that it's only going to be the "page title", but I am not sure, I'm fine with that being the definition of such pages' headers. If it's more expensive to define now and revise later (as more pages like this might come up in the future), then I'm fine with a temporary solution where the title (header) and the indented content is readable. I understand that the "startword, n. ...." is not part of the header, I'm not sure what the solution is here. Maybe shift the body further down (below the issues purple box)? 

---------------------
- @gwijthoff thank you :). I like "|" too! It just looks slightly odd when it comes before the bullets in Board of Advisors. It might look better if the names don't appear in bullets but rather listed after the "|" on the same line separated with commas. But I don't feel strongly and what I just suggested might actually mess up the layout on mobile and it's more risky ...so it's fine as is unless any of you @rlskoeser @thatbudakguy have other suggestions. 

- Grant's suggestion about making h2 makes sense to me. I defer to @thatbudakguy.

-  @gwijthoff, thanks for explaining about the "masthead". I'm just not sure if that term is inclusive enough for the general public. But, I don't feel strongly, this can be revised later in some way. 

- @rlskoeser What do you mean by a "definition list"?

**Comment by thatbudakguy** [comment]
ok, I've added a solution that I think will work going forward.

- the about page stores the content in its header (the definition of _startword_) as its summary, via a [yaml front matter summary](https://gohugo.io/content-management/summaries/#front-matter-summary). you can see [what that looks like in the markdown file](https://github.com/Princeton-CDH/startwords/blame/master/content/about.md#L4-L8).
- the default page template (which is used for the about page) renders the `<h1>` along with the page's summary in a special container that has different margins. I used the same margin scaling as the issue preview at the top of the homepage.
- the homepage now finds the about page and renders its summary at the bottom, instead of having to duplicate the _startword_ definition text in the homepage content file. If you edit the about page summary, that change will be reflected on the homepage.

@gissoo let me know if the margin scaling looks right to you (check desktop as well). @gwijthoff let me know if the above description doesn't make sense. it's a little bit of an abuse of the "summary" function, but I think if we're aware of it it will work OK.

**Comment by gissoo** [comment]
@thatbudakguy thank you for your work on this and for coming up with this solution!! This is great!! The header's margins are incorrect on desktop, they should align with the body. They look perfect on mobile.

**Comment by gwijthoff** [comment]
@thatbudakguy I like this solution. Agreed with @gissoo that the header's margins look great on mobile now, but that they're too roomy on desktop. It looks awkward with how far over to the right "About" and the Startwords dictionary definition are placed.

**Comment by thatbudakguy** [comment]
ok, updated default pages to have the same header margins as article pages.

**Comment by gissoo** [comment]
@thatbudakguy thank you so much!! This looks great!
